docs/typelibrary/utils/conversion_struct/forwarding/ST08B_TO_BOOLS.md

```
@@ -0,0 +1,45 @@
+# ST08B_TO_BOOLS
+
+![ST08B_TO_BOOLS](https://github.com/user-attachments/assets/5ab5b2e2-dd67-448f-8306-12ec64fd03c5)
+
+* * * * * * * * * *
+## Einleitung
+Der Funktionsblock `ST08B_TO_BOOLS` dient zur Konvertierung eines `ST08B`-Datentyps in acht einzelne `BOOL`-Werte. Dies ermöglicht die einfache Extraktion einzelner Bits aus einem Byte.
+
+## Schnittstellenstruktur
+
+### **Ereignis-Eingänge**
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)